app/code/Magento/CatalogImportExport/Model/Import/Product/Option.php

```diff
@@ -2098,6 +2093,7 @@ protected function _parseCustomOptions($rowData)
             $optionValueParams = explode($this->_productEntity->getMultipleValueSeparator(), $optionValue);
             foreach ($optionValueParams as $nameAndValue) {
                 $nameAndValue = explode('=', $nameAndValue);
+                // @phpstan-ignore-next-line
```

Comment:
Static test failure seems to be valid. call to `explode` with empty string returns array with single element( which is not empty). Is there a case when explode will return "empty" value ?
Could you please do corresponding changes ?

Reply:
Hello @xmav 
I think we need to refactor it with additional test coverage
We are going to do it tomorrow

Reply:
@xmav I removed the unnecessary 'if'.
@andrewbess  as discussed, the complete refactoring with a new test cases will be implemented in the future due to complexity.